src/main/java/io/kestra/plugin/hightouch/connections/Sync.java

```diff
@@ -0,0 +1,213 @@
+package io.kestra.plugin.hightouch.connections;
```

Comment:
Better to move this task to the hightouch package as there is only one task.
subpackage are used only when you want to have multiple group of tasks in a plugin.